jest.setup.js

```diff
@@ -0,0 +1,30 @@
+// How to catch unhandled errors when testing:
+// Reference:
+// https://chris-washington-dev.medium.com/vue-jest-properly-catch-unhandledpromiserejectionwarning-and-vue-warn-errors-in-jest-unit-tests-fcc45269146b
+
+// you need this to reformat the console.error
+import { format } from 'util';
+
+process.on('unhandledRejection', (error) => {
+  // Will print "unhandledRejection err is not defined"
+  console.error('unhandledRejection', error.message);
```

Comment:
Given what you have below, it sounds like you want to `console.error` and… throw? Why all the magic?

Reply:
This was in the code I source at the top of that file. I don't know about the unhandledRejection bit, but I definitely have seen cases where we weren't handling failed promises, and also cases where vue was emitting '[VueSlider error]' messages to the console but we want to verify that the error has been detected when testing. If you comment out all of `jest.setup.js` and run the tests you'll see the error messages in the shell, but they're lost -- I didn't find a way to spy on `console.error` (although that would work).